server.py

```diff
@@ -148,6 +148,43 @@ def getUser():
 	return jsonify(info), 200
 
 
+@app.route('/update/password', methods=['PUT'])
+def changePassword():
+	csrf.protect()
+
+	login_code = session.get('login', None)
+	if login_code is None:
+		return forbidden()
+
+	login = LoginCode.get_by_code(login_code)
+	if login is None:
+		return forbidden()
+
+	json = request.json
+	if json is None:
+		return 'Missing json data', 400
+
+	old = json.get('pass_old', None)
+	new1 = json.get('pass_new', None)
+	new2 = json.get('pass_new_conf', None)
+
+	if old is None or new1 is None or new2 is None:
+		return 'Missing fields', 400
```

Comment:
We should also handle "", 0 length passwords. Perhaps we should have an IsValidPassword() function in utils. Configurable with whatever requirements the site wants for passwords.

Reply:
We might want to drop this here and make an issue. Since this affects more places than just this.